src/PrestaShopBundle/Controller/Admin/Sell/Order/CartController.php

```diff
@@ -313,10 +377,88 @@ public function addProductAction(Request $request, int $cartId): JsonResponse
         }
     }
 
+    /**
+     * @AdminSecurity("is_granted('update', request.get('_legacy_controller')) || is_granted('create', 'AdminOrders')")
+     *
```

Comment:
Can we add here a quick explanation (to help people understand what I did):
"Modifying a price for a product in the cart is actually performed by using generated specific prices, that are used only for this cart and this product."

Reply:
Now I understand a bit more how some shops can have thousands of specific prices